TrackletAlgorithm/VMRouterTop.cc

```diff
@@ -27,12 +27,12 @@ void VMRouterTop(BXType bx,
 // Variables that are specified with regards to the test TestBenches
```

Comment:
Same comment as in MatchCalculatorTop.cpp: I wonder if we should use the inline pragma here to save a clock cycle otherwise wasted with the extra function call.